projects/ngrx.io/tools/example-zipper/exampleZipper.js

```diff
@@ -144,8 +144,8 @@ class ExampleZipper {
 
     let gpaths = json.files.map((fileName) => {
       fileName = fileName.trim();
-      if (fileName.substr(0, 1) === '!') {
-        return '!' + path.join(exampleDirName, fileName.substr(1));
+      if (fileName.substring(0, 1) === '!') {
```

Comment:
this could be simplify ?
```suggestion
      if (fileName[0] === '!') {
```

Reply:
Yes sure. Good catch Thanks